### PR TITLE
Make lightning have frag angle properties

### PR DESCRIPTION
**Pull request by MEEPofFaith**
Adds `lightningCone`, which is the full angle of where the lightning direction can be, and `lightningAngle` which is the angle compared to the bullet that the center of the lightning cone is.

### CODE REVIEW

File: core/src/mindustry/entities/bullet/BulletType.java [hunk 1]
```diff
@@ -181,7 +184,8 @@ public void hit(Bullet b, float x, float y){
         }
 
         for(int i = 0; i < lightning; i++){
-            Lightning.create(b, lightningColor, lightningDamage < 0 ? damage : lightningDamage, b.x, b.y, Mathf.random(360f), lightningLength + Mathf.random(lightningLengthRand));
+            float litA = b.rotation() + Mathf.range(lightningCone/2) + lightningAngle;
```
Comment (Anuken) [inline core/src/mindustry/entities/bullet/BulletType.java:187]: This should go in the function itself for consistency. No extra variable.

Comment (MEEPofFaith) [reply core/src/mindustry/entities/bullet/BulletType.java:187]: Is there a button for accepting suggested requests, or is it committed already?

Comment (Anuken) [reply core/src/mindustry/entities/bullet/BulletType.java:187]: There is no suggested code change or commit. It's a verbal request.

Comment (MEEPofFaith) [reply core/src/mindustry/entities/bullet/BulletType.java:187]: Ok

**Pull request merged**